src/coreclr/src/inc/crosscomp.h

```diff
@@ -198,14 +196,14 @@ typedef struct _T_DISPATCHER_CONTEXT {
     PUCHAR NonVolatileRegisters;
```

Comment:
If it is just required to make this compile, the pointers like `HistoryTable` in cross-targeting version of this structure may be replaced with `TADDR` or `PVOID`. It do not think anybody will access them in the DAC.

Reply:
> do not think anybody will access them in the DAC.

@jkotas This seems to be related to native unwinding.

@mikem8361 Had to port unwind to the out of process DAC to support unwinding of native frames in a dump.  I have native unwinding disabled temporarily, but we will eventually want to re-enable it in the Cross OS DAC. I don't want to create deferred work.

Reply:
> This seems to be related to native unwinding.

It is specific to Windows exception handling internals. As far as I can tell, we do not use  `UNWIND_HISTORY_TABLE` anywhere in the runtime. So I believe that you should be able to delete the definition of `UNWIND_HISTORY_TABLE` and replace it `PUNWIND_HISTORY_TABLE HistoryTable;` in `_T_DISPATCHER_CONTEXT` struct with just `PTR_VOID HistoryTable;`.

Reply:
I may have gone too far.  I ripped it out from the runtime entirely.  I can revert the palrt.h changes ….

Reply:
:shipit: